.pre-commit-config.yaml

```diff
@@ -5,3 +5,4 @@ repos:
     - id: black
       language_version: python3
       require_serial: true
+      exclude: 'vendored|\_vispy'
```

Comment:
this line should be deleted

Reply:
I needed to add this line because otherwise the folders would get reformatted as detailed in #335 - I am understanding this wrong?

I'm not sure why its `\_vispy` instead of `_vispy` I don't remember doing that

Reply:
removed from this PR